day_10/task_1/tests/pipe_maze_tests.cpp

```diff
@@ -5,5 +5,16 @@ TEST(PipeMazeToolsShould, FindTheStartingPoint)
 {
     const PipeMaze pipe_maze = {"-L|F7", "7S-7|", "L|7||", "-L-J|", "L|-JF"};
 
-    ASSERT_EQ(find_starting_point(pipe_maze), (std::make_pair<int, int>(1, 1)));
+    ASSERT_EQ(find_starting_point(pipe_maze), MazePoint(1, 1));
```

Comment:
If you refactor this line, then I think for consistency you should also refactor the function find_the_starting_point and apply MazePoint, where it is applicable.